src/Rules/NoReturnSetterMethodRule.php

```diff
@@ -63,15 +67,15 @@ public function processNode(Node $node, Scope $scope): array
         }
 
         $classMethodName = $node->name->toString();
-        if ($classMethodName === 'setUp') {
+        if (strpos($classMethodName, 'setUp') === 0) {
```

Comment:
IMHO `setUp` is it's own prefix that is not confused with a "normal" setter, so i think this rule should skip all methods prefixed with `setUp`